keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -258,7 +259,19 @@ func (runtime *CRIRuntime) PullImages(images []string) error {
 // The same way as func (runtime *DockerRuntime) CopyResources
 func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]string) error {
 	psc := &runtimeapi.PodSandboxConfig{
-		Metadata: &runtimeapi.PodSandboxMetadata{Name: KubeEdgeBinaryName},
+		Metadata: &runtimeapi.PodSandboxMetadata{
+			Name:      KubeEdgeBinaryName,
+			Namespace: constants.SystemNamespace,
+		},
+		Linux: &runtimeapi.LinuxPodSandboxConfig{
+			SecurityContext: &runtimeapi.LinuxSandboxSecurityContext{
+				NamespaceOptions: &runtimeapi.NamespaceOption{
+					Network: runtimeapi.NamespaceMode_POD,
+					Pid:     runtimeapi.NamespaceMode_CONTAINER,
```

Comment:
The pid config can be removed here. FYI: #5144

Reply:
> The pid config can be removed here. FYI: #5144

Done, thanks